release.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Perform a docker-machine release
+# GITHUB_TOKEN=XXXXX ./release.sh 0.X.X
+
+#GITHUB_USER=docker
+GITHUB_USER=jeanlaurent
+GITHUB_REPO=machine
+
+
+function display {
+    echo "🐳   $1"
```

Comment:
Won't this cause issues if someone uses the script from Windows or Linux ?  e.g. inside of a container